apps/console/src/features/users/components/wizard/invite-parent-org-user-wizard.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    IdentifiableComponentInterface,
+    TestableComponentInterface
+} from "@wso2is/core/models";
+import { Heading, LinkButton, PrimaryButton, useWizardAlert } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Grid, Modal } from "semantic-ui-react";
+import { UserManagementConstants } from "../../constants";
+import { InviteParentOrgUser } from "../guests/pages/invite-parent-org-user";
+
+/**
+ * Interface for the Invite Parent Org User Wizard.
+ */
+interface InviteParentOrgUserWizardProps extends IdentifiableComponentInterface, TestableComponentInterface {
+    closeWizard: () => void;
+    /**
+     * The Callback to trigger on user invite success.
+     */
+    onUserInviteSuccess?: () => void;
+}
+
+export const InviteParentOrgUserWizard: FunctionComponent<InviteParentOrgUserWizardProps> = (
+    props: InviteParentOrgUserWizardProps
+): ReactElement => {
+
+    const {
+        closeWizard,
+        onUserInviteSuccess,
+        [ "data-testid" ]: testId
```

Comment:
Let's use componentid as using testid has been deprecated.

```suggestion
        [ "data-componentid" ]: componentId
```

Reply:
Changed